trin-storage/src/versioned/id_indexed_v1/config.rs

```diff
@@ -11,10 +11,7 @@ use crate::{
 };
 
 /// The fraction of the storage capacity that we should aim for when pruning.
-const TARGET_CAPACITY_FRACTION: f64 = 0.9;
-
-/// The fraction of the storage capacity that we need to pass in order to start pruning.
-const PRUNING_CAPACITY_THRESHOLD_FRACTION: f64 = 0.95;
+const TARGET_CAPACITY_FRACTION: f64 = 0.95;
```

Comment:
Maybe add a note here explaining why it's set to `95%` & what side effects to expect if someone wants to change it in the future